proto/spire/plugin/server/credentialcomposer/v1/credentialcomposer.proto

```diff
@@ -0,0 +1,179 @@
+// The CredentialComposer plugin influences the attributes of X.509
```

Comment:
nit: Should this be the service's comment?

Reply:
:+1: